pinot-core/pom.xml

```diff
@@ -133,6 +133,10 @@
       <groupId>io.netty</groupId>
       <artifactId>netty</artifactId>
     </dependency>
+    <dependency>
```

Comment:
should we not remove all other netty-xyz.jar  if we add netty-all?

Reply:
We only have netty and netty-all right?